### ISSUE TITLE
Gradle Plugin Fails on Windows

### ISSUE THREAD

**Issue by jamesward** [apple/pkl#608]
With a few projects which use the Pkl Gradle plugin that work on Linux, they fail on Windows with:
```
–– Pkl Error ––
Refusing to load module `C:/Users/james/Desktop/projects/hello-pkl/src/hello.pkl` because it does not match any entry in the module allowlist (`--allowed-modules`).
```

If I modify the `pkl` Gradle config to be:
```
pkl {
    evaluators {
        register("evalPkl") {
            allowedModules = listOf("")
            sourceModules.add(file("src/hello.pkl"))
            outputFile = layout.buildDirectory.file("hello.yaml")
            outputFormat = "yaml"
        }
    }
}
```
(overriding the `allowedModules` to be more permissive) I get:
```
–– Pkl Error ––
I/O error loading module `C:/Users/james/Desktop/projects/hello-pkl/src/hello.pkl`.
MalformedURLException: unknown protocol: c
```

So this looks like some weirdness with URIs and the Windows `C:` stuff.

**Comment by bioball** [comment]
This was fixed in https://github.com/apple/pkl/pull/403.

For now, the workaround is to give Pkl a URI. For example:

```kt
sourceModules.add(file("src/hello.pkl").toURI())
```

**Comment by jamesward** [comment]
Oh awesome! Thank you!

**Comment by jamesward** [comment]
This might be fixed in #403 but even when turning the file into a URI, I still get the error:
```
Refusing to load module `C:/Users/james/Desktop/projects/hello-pkl/src/hello.pkl` because it does not match any entry in the module allowlist (`--allowed-modules`).
```

**Comment by jamesward** [comment]
Any other workaround ideas? Or any easy way to test the changes in #403 on Windows?

**Comment by bioball** [comment]
This was just fixed in release 0.26.3! Give that a shot and let us know if you have any issues.

**Comment by jamesward** [comment]
It works now! Thank you!
